pkg/asset/machines/machineconfig/disks.go

```diff
@@ -0,0 +1,173 @@
+package machineconfig
+
+import (
+	"bytes"
+	"fmt"
+	"regexp"
+	"strings"
+	"text/template"
+
+	igntypes "github.com/coreos/ignition/v2/config/v3_2/types"
```

Comment:
this does remind me we should probably update the installer ignition default type (MCO is using 3.5)